tests/integration/defs/accuracy/test_llm_api_autodeploy.py

```
@@ -719,6 +719,72 @@ def test_mtp(self, world_size):
         print_memory_usage("after evaluation")
 
 
+class TestNemotronUltraV3(LlmapiAccuracyTestHarness):
+    MODEL_NAME = "nvidia/Nemotron-Ultra-V3"
+    CONFIG_YAML = str(
+        Path(get_llm_root()) / "examples" / "auto_deploy" / "model_registry" /
+        "configs" / "ultra_v3.yaml")
+    MODEL_PATHS = {
+        "nvfp4": hf_id_to_local_model_dir("nvidia/Nemotron-Ultra-V3-NVFP4"),
+    }
```

Ruff (0.15.12)

[warning] Mutable default value for class attribute

(RUF012)